applications/nrf_desktop/doc/battery_charger.rst

```diff
@@ -0,0 +1,56 @@
+.. _battery_charger:
+
+Battery charger module
+######################
+
+The ``battery_charger`` module is responsible for battery charging control.
+
+Module Events
+*************
+
++------------------+---------------------+---------------------+-------------------------+------------------+
+| Source Module    | Input Event         | This Module         | Output Event            | Sink Module      |
++==================+=====================+=====================+=========================+==================+
+|                  |                     | ``battery_charger`` | ``battery_state_event`` | :ref:`led_state` |
++------------------+---------------------+                     +-------------------------+------------------+
+| :ref:`usb_state` | ``usb_state_event`` |                     |                         |                  |
++------------------+---------------------+---------------------+-------------------------+------------------+
+
+Configuration
+*************
+
+The module implemented in ``battery_charger.c`` uses the Zephyr :ref:`zephyr:gpio` driver to control and monitor battery charging. For this reason, you should set :option:`CONFIG_GPIO` option.
+
+By default, the module is disabled and the ``CONFIG_DESKTOP_BATTERY_CHARGER_NONE`` option is selected.
+Set the option ``CONFIG_DESKTOP_BATTERY_CHARGER_DISCRETE`` to enable the module.
+
+The following module configuration options are available:
+* ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PIN`` - pin to which the charge status output (CSO) from the charger is connected.
+* CSO pin pull:
+
+  * ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PULL_NONE`` - disabled (default setting)
+  * ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PULL_UP`` - up
+  * ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PULL_DOWN`` - down
+
+* ``CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_PIN`` - pin that enables the battery charging.
+* ``CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_INVERSED`` - inverse charging enable signal.
```

Comment:
```suggestion
* ``CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_INVERSED`` - option for signal that enables inverse charging.
```

Reply:
It is option used to inverse charging enable signal (negate the binary signal). It does not inverse charging.